src/server/db_slice.cc

```diff
@@ -337,10 +338,10 @@ void DbSlice::AutoUpdater::Run() {
 
   // Make sure that the DB has not changed in size since this object was created.
   // Adding or removing elements from the DB may invalidate iterators.
-  CHECK_EQ(fields_.db_size, fields_.db_slice->DbSize(fields_.db_ind))
+  DCHECK_EQ(fields_.db_size, fields_.db_slice->DbSize(fields_.db_ind))
       << "Attempting to run post-update after DB was modified";
 
-  CHECK_EQ(fields_.deletion_count, fields_.db_slice->deletion_count_)
+  DCHECK_EQ(fields_.deletion_count, fields_.db_slice->deletion_count_)
```

Comment:
Not for this PR, but perhaps we should swap the order of these checks.
This (second) one is easier to debug, and also a subset of the above, so it would be good to know if it triggered